dashboard-app/src/lib/api/index.ts

```diff
@@ -1,10 +1,12 @@
 import _axios, { AxiosRequestConfig, Method } from 'axios';
-import { Api } from '../../../../api/src/api/v1/types/api';
+import { Api } from '../../../../api/src/api/v1.1/types/api';
 import qs from 'qs';
 
+const API_VERSION = 'v1.1';
+
 const baseURL = process.env.REACT_APP_API_HOST_DOMAIN ?
```

Comment:
Can you move the `?` to the next line?

It would be useful to add a more specific linting config to all our apps. I am using `airbnb` on the native one.